quill-orientdb/src/test/scala/io/getquill/context/orientdb/orientdb.scala

```diff
@@ -0,0 +1,66 @@
+package io.getquill.context.orientdb
+
+import com.orientechnologies.orient.client.remote.OServerAdmin
+import com.orientechnologies.orient.core.db.OPartitionedDatabasePool
+import com.orientechnologies.orient.core.metadata.schema.OSchemaProxy
+import io.getquill._
+
+object orientdb {
+  private val dbUrl = "remote:orientdb:12424"
+  private val databaseName = "GratefulDeadConcerts"
+  private val databaseType = "document"
+  private val storageMode = "memory"
+  private val username = "root"
+  private val password = "root"
```

Comment:
`12424` is the port number on Docker host to which internal `2424` port is mapped. It's useful when you want to reach this port from outside (from a different machine). But here internal port must be used because orientdb container is linked to sbt container.